fmriprep/workflows/bold/base.py

```diff
@@ -357,7 +357,7 @@ def init_func_preproc_wf(
             fmaps.append({'suffix': 'syn'})
 
         # Short circuits: (True and True and (False or 'TooShort')) == 'TooShort'
-        run_stc = ("SliceTiming" in metadata and
+        run_stc = (metadata.get("SliceTiming", False) and
```

Comment:
If `None` doesn't work, `[]` isn't going to work.
```suggestion
        run_stc = (bool(metadata.get("SliceTiming")) and
```